src/drive/web/modules/filelist/FileListBody.jsx

```diff
@@ -8,19 +9,24 @@ import { isSelectionBarVisible } from 'drive/web/modules/selection/duck'
 
 import styles from 'drive/styles/filelist.styl'
 
-const FileListBody = ({ className, children }) => {
+const FileListBody = ({ className, children, isFabActive }) => {
```

Comment:
This `isFabActive` is here because we don't display the fab in every views we have in Drive? If this is not that, can you explain me when the fab can not be displayed?

Reply:
Oui le FileListBody est également utilisé dans la modale de déplacement